pages/community/overview.mdx

```diff
@@ -23,7 +23,8 @@ News are broadcasted on this site's blog, on Bluesky and X. Some extra, less imp
 
 ## Articles
 
-- [Getting rid of your dead code in ReScript](https://dev.to/zth/getting-rid-of-your-dead-code-in-rescript-3mba)
+- [Community Blog](https://rescript-lang.org/blog/community)
```

Comment:
Can you make that link relative?

Reply:
Should I update the other links to also be relative?

Reply:
Done.